hack/unwanted-dependencies.json

```diff
@@ -181,15 +174,17 @@
         "github.com/Microsoft/hcsshim"
       ],
       "golang.org/x/exp": [
+        "github.com/Microsoft/hcsshim",
         "github.com/antlr4-go/antlr/v4",
+        "github.com/cilium/ebpf",
         "github.com/google/cel-go",
         "k8s.io/mount-utils"
       ],
       "google.golang.org/appengine": [
         "github.com/prometheus/client_golang"
       ],
       "google.golang.org/genproto": [
-        "github.com/containerd/ttrpc",
+        "github.com/Microsoft/hcsshim",
```

Comment:
I'm really surprised to see something new pick up a dep here... we're trying *really* hard to drop this dep entirely (https://github.com/kubernetes/kubernetes/issues/113366)

We've already carved out all the generic proto libs that attracted use... what is hcsshim using in the root genproto dep? can we get that updated to no longer have this dep?